### PR TITLE
README updates

### LINKED ISSUE #88: Cleanup README links for `mc-sgx-*-sys*` crates.

### PR DESCRIPTION
**Pull request by jcape**
Fixes #88

### CODE REVIEW

**Comment by jcape** [comment]
Current dependencies on/for this PR:
* main
  * **PR #94** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/94" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/94?utm_source=stack-comment).

File: capable/Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+description = "Rust wrappers for the `sgx_capable` library."
+
+categories = ["external-ffi-bindings", "hardware-support"]
+description = "FFI linkage for the `sgx_capable` library."
+edition = "2021"
+keywords = ["sgx"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/mobilecoinfoundation/sgx"
+rust-version = "1.62.1"
```
Comment (nick-mobilecoin) [inline capable/Cargo.toml:20]: :upside_down_face: Looks like most of these are repeats.  `capable` is not in the root `Cargo.toml` workspace, if it gets added then these duplicates show up as cargo errors.
```suggestion
keywords = ["sgx"]
```

File: capable/Cargo.toml [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
 rust-version = "1.62.1"
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/mobilecoinfoundation/sgx"
-description = '''
-This crate exposes an idiomatic rust interface to the SGX SDK's `libsgx_capable` libraries.
-'''
+description = "Rust wrappers for the `sgx_capable` library."
+
+categories = ["external-ffi-bindings", "hardware-support"]
```
Comment (nick-mobilecoin) [inline capable/Cargo.toml:13]: :upside_down_face: Looking at https://crates.io/categories it seems that `external-ffi-bindings` are the ffi crates, while `api-bindings` are the idiomatic crates.
```suggestion
categories = ["api-bindings", "hardware-support"]
```

File: capable/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+-->[![Crates.io][crate-image]][crate-link]<!--
+-->[![Docs Status][docs-image]][docs-link]<!--
+-->[![Dependency Status][deps-image]][deps-link]
+
+Idiomatic rust for the `sgx_capable` library.
+
+[chat-image]: https://img.shields.io/discord/844353360348971068?style=flat-square
+[chat-link]: https://mobilecoin.chat
+[license-image]: https://img.shields.io/crates/l/mc-sgx-capable?style=flat-square
+[target-image]: https://img.shields.io/badge/target-x86__64--unknown--linux--gnu-orange?style=flat-square
```
Comment (nick-mobilecoin) [inline capable/README.md:15]: :wrench: Should this be x86_64, it looks like capable is used for checking and enable SGX behavior so it's client side.  If it should be unknown then `capable/sys/README.md` should probably be updated n'sync.
```suggestion
[target-image]: https://img.shields.io/badge/target-x86__64-blue?style=flat-square
```

File: core/types/src/lib.rs [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 //! Rust wrappers for SGX types
```
Comment (nick-mobilecoin) [inline core/types/src/lib.rs:3]: :upside_down_face: Thinking this comment can go away with the readme include
```suggestion
```

File: dcap/sys/types/src/lib.rs [hunk 5]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2022 The MobileCoin Foundation
 //! Rust FFI types for the SGX SDK DCAP libraries.
```
Comment (nick-mobilecoin) [inline dcap/sys/types/src/lib.rs:2]: :upside_down_face: Thinking this can go away with the readme include
```suggestion
```

**Review by nick-mobilecoin — APPROVED** [review]

**Review by awygle — APPROVED** [review]

**Pull request merged**